testcases/kernel/controllers/cpuctl/cpuctl_def_task04.c

```diff
@@ -80,7 +80,7 @@ extern void cleanup()
 
 volatile int timer_expired = 0;
 
-int main(int argc LTP_ATTRIBUTE_UNUSED, char *argv[] LTP_ATTRIBUTE_UNUSED)
+int main(void)
 {
 
 	int test_num;
```

Comment:
Can you squash all the "Make unused parameters explicit" and "Change main to main(void)" commits into one?